cdap-docs/admin-manual/source/security.rst

```diff
@@ -450,16 +449,17 @@ will use ``https`` instead of ``http``.
 
     {"auth_uri":["http://localhost:10009/token"]}
 
-- Submit a username and password to one of the URIs to obtain an ``AccessToken``::
+- Submit a username and password to one of the authentication URIs (``<auth-uri>``) to
+  obtain an ``AccessToken``::
 
-    GET -u <username>:<password> <auth-uri>
+    GET <auth-uri> <username>:<password>
```

Comment:
I don't think we need the curl at all to define a REST endpoint. curl is only one way in which people make REST calls. We should describe exactly what you suggest: the HTTP method, the URL, the accepted headers and the expected request body.